spectacles/validators/data_test.py

```diff
@@ -93,40 +93,44 @@ async def get_tests(self, project: Project) -> List[DataTest]:
 
     async def validate(self, tests: List[DataTest]) -> List[DataTestError]:
         data_test_errors: List[DataTestError] = []
-
-        async def run_test(test: DataTest) -> None:
-            results = await self.client.run_lookml_test(
-                test.project_name, model=test.explore.model_name, test=test.name
-            )
-            test.explore.queried = True
-            result = results[0]  # For a single test, list with length 1
-
-            if result["success"]:
-                test.passed = True
-                test.explore.successes.append(
-                    {
-                        "model": test.explore.model_name,
-                        "explore": test.explore.name,
-                        "metadata": {
-                            "test_name": result["test_name"],
-                            "lookml_url": test.lookml_url,
-                            "explore_url": test.explore_url,
-                        },
-                    }
+        query_slot = asyncio.Semaphore(
+            15
```

Comment:
Let's define this `15` as a constant at the top of the module.

Reply:
Will do!